brink/the-bitcoin-development-podcast/discussing-0-21-0-bitcoin-core-vulnerability-disclosures.md

```diff
@@ -1,60 +1,61 @@
+date: '2024-07-31'
+tags:
+  - 'bitcoin-core'
+speakers:
+  - 'Gloria Zhao'
+  - 'Niklas Gögge'
 categories: []
-summary: "Brink engineers Gloria Zhao and Niklas Gögge talk through the recently disclosed Bitcoin Core 0.21.0 vulnerabilities."
-date: 2024-07-31
+summary: 'Brink engineers Gloria Zhao and Niklas Gögge talk through the recently disclosed Bitcoin Core 0.21.0 vulnerabilities.'
 additional_resources:
-  - title: Episode 4
-    url: https://brink.dev/podcast/4-bitcoin-core-pre-21-disclosures/
-  - title: Disclosure of remote crash due to addr message spam
-    url: https://bitcoincore.org/en/2024/07/31/disclose-addrman-int-overflow/
+  - title: 'Episode 4'
+    url: 'https://brink.dev/podcast/4-bitcoin-core-pre-21-disclosures/'
+  - title: 'Disclosure of remote crash due to addr message spam'
+    url: 'https://bitcoincore.org/en/2024/07/31/disclose-addrman-int-overflow/'
   - title: 'Bitcoin Core #22387'
-    url: https://github.com/bitcoin/bitcoin/pull/22387
-  - title: Disclosure of the impact of an infinite loop bug in the miniupnp dependency
-    url: https://bitcoincore.org/en/2024/07/31/disclose-upnp-oom/
+    url: 'https://github.com/bitcoin/bitcoin/pull/22387'
+  - title: 'Disclosure of the impact of an infinite loop bug in the miniupnp dependency'
+    url: 'https://bitcoincore.org/en/2024/07/31/disclose-upnp-oom/'
   - title: 'Bitcoin Core #20421'
-    url: https://github.com/bitcoin/bitcoin/pull/20421
+    url: 'https://github.com/bitcoin/bitcoin/pull/20421'
 ---
 ## Introduction
 
-Speaker 0: 00:00:00
+Gloria Zhao: 00:00:00
 
 Hello, Niklas.
 
-Speaker 1: 00:00:02
+Niklas Gögge: 00:00:02
 
 Hi, Gloria.
 We're here to talk about the next batch of disclosures for Bitcoin Core.
 And this time there's only two bugs and they were fixed in version 22.
-So that means 21, version 21 was still vulnerable to these two bugs.
+So that version 21 was still vulnerable to these two bugs.
 If you're running version 21, you should upgrade.
-I mean, you can listen to this podcast and decide for yourself if you want to upgrade, but my recommendation, our recommendation would be that you upgrade.
+I mean, you can listen to this podcast and decide for yourself if you want to upgrade, but our recommendation would be that you upgrade.
 Next month, we'll be doing bugs for version 22.
 So ideally, you upgrade to the latest maintained versions, which the lowest one currently is 25.
 So you should upgrade to 25, 26, or 27.
 
-Speaker 0: 00:00:50
+Gloria Zhao: 00:00:50
 
 Right.
 So yeah, spoiler, but don't just upgrade to 22.
 
-Speaker 1: 00:00:58
+Niklas Gögge: 00:00:58
 
 Yeah, because you're just doing the same thing every month.
 
-Speaker 0: 00:01:02
+Gloria Zhao: 00:01:02
 
-OK, great.
+Great.
 So why don't we dive into the disclosures.
 
 ## Background on Bitcoin peer-to-peer address relay
 
-Speaker 0: 00:01:07
```

Comment:
This is like the 3rd time that I tell you not to remove timestamps

Reply:
Sometimes you tell me to remove sometimes not to so I don't know which is which. The last time you told me to remove because the speaker is continuing after the topic title.